tests/conftest.py

```diff
@@ -237,3 +244,56 @@ def mock_get_keyvault_key(self, key_name, key_vault_name):  # noqa: ARG001
 #         return [MockStorageAccountKey("key1"), MockStorageAccountKey("key2")]
 
 #     monkeypatch.setattr(AzureApi, "get_storage_account_keys", mock_get_storage_account_keys)
+
```

Comment:
I know it's not part of this PR, but can we clean up the commented-out lines 237-246 here?